ls8/cpu.c

```diff
@@ -64,4 +75,7 @@ void cpu_run(struct cpu *cpu)
 void cpu_init(struct cpu *cpu)
 {
   // TODO: Initialize the PC and other special registers
+  cpu->pc = 0;
+  memset(&(cpu->reg), 0, 8 * sizeof(unsigned char));
+  memset(&(cpu->ram), 0, 256 * sizeof(unsigned char));
```

Comment:
Great job initializing & figuring out how to apply memset! Another way of accomplishing this via calloc is "cpu->ram = calloc(256, sizeof(unsigned char));"